quota/etcd/quotaapi/quota_server.go

```diff
@@ -16,21 +16,54 @@
 package quotaapi
 
 import (
+	"github.com/coreos/etcd/clientv3"
 	"github.com/golang/protobuf/ptypes/empty"
 	"github.com/google/trillian/quota/etcd/quotapb"
+	"github.com/google/trillian/quota/etcd/storage"
+	"github.com/google/trillian/quota/etcd/storagepb"
 	"golang.org/x/net/context"
+	"google.golang.org/genproto/protobuf/field_mask"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
 var errNotImplemented = status.Error(codes.Unimplemented, "not implemented")
 
 // Server is a quotapb.QuotaServer implementation.
```

Comment:
Done (although I added a single word). I'll add proper documentation in form of a README once this is integrated, but I'd like to leave that as the last step.